### DISCUSSION TITLE
Thanks so much!

### DISCUSSION THREAD

**Discussion by Jayd603** [MrNaif2018/bitcart#265] [General]
I just did a successful test on my WHMCS install and it worked great.  I'm really impressed with this project so far and think it will get way more adoption once people learn it exists.  : - )

I donated but could not set the amount to donate.  I'm assuming I can over-pay the invoice next time?

Also, I offer hosting services at my business and might start offering BitcartCC but i'd need to spend some time becoming more familiar with it.

Anyway, great work to anyone who contributed!

**Comment by MrNaif2018** [comment]
Thank you very much for your kind words!

> it will get way more adoption once people learn it exists. : - )

Definitely! Just need to find a way to make people know about it (:

As for the donation, the donate button on the site is configured to create a 5usd donation, if you want to play with the store POS and customize amounts a bit (though we don't have custom amounts yet), you can do via store POS's cart:
https://store.bitcartcc.com

We are open to any suggestions on how to improve the project! Thanks!
